### PR TITLE
Updated wording of NFS limitation

### PR DESCRIPTION
**Pull request by drothery-edb**
## What Changed?

EPAS:

- Moved limitations from Supported Platforms topic to a Limitations topic under Release notes
- Rewrote NFS limitation
- Fixed 14 version number and upstream merge links in rel notes
- Removed NFS limitation from Limitations topics under Upgrade Guide 
- Renamed the Limitations topic under the Upgrade Guide to Upgrade Considersations
- Deleted Supported Platforms topic under Upgrade Guide
- Streamlined and clarified intro sentences to limitation lists

### CODE REVIEW

File: product_docs/docs/epas/10/epas_upgrade_guide/02_limitations.mdx [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
-title: "Limitations"
+title: "Upgrade Considerations"
 
 ---
 
-The following limitations apply to EDB Postgres Advanced Server:
+The following limitations apply to upgrading EDB Postgres Advanced Server:
 
--   The `pg_upgrade` utility cannot upgrade a partitioned table if a foreign key refers to the partitioned table.
```
Comment (kelpoole) [inline product_docs/docs/epas/10/epas_upgrade_guide/02_limitations.mdx:8]: Why was this removed?

Comment (drothery-edb) [reply product_docs/docs/epas/10/epas_upgrade_guide/02_limitations.mdx:8]: I removed it by mistake - restored now

**Review by kelpoole — DISMISSED** [review]
